cluster-autoscaler/provisioningrequest/checkcapacity/provisioningclass.go

```diff
@@ -139,7 +143,13 @@ func (o *checkCapacityProvClass) checkcapacity(unschedulablePods []*apiv1.Pod, p
 	err, cleanupErr := clustersnapshot.WithForkedSnapshot(o.context.ClusterSnapshot, func() (bool, error) {
 		st, _, err := o.schedulingSimulator.TrySchedulePods(o.context.ClusterSnapshot, unschedulablePods, scheduling.ScheduleAnywhere, true)
 		if len(st) < len(unschedulablePods) || err != nil {
-			conditions.AddOrUpdateCondition(provReq, v1.Provisioned, metav1.ConditionFalse, conditions.CapacityIsNotFoundReason, "Capacity is not found, CA will try to find it later.", metav1.Now())
+			if noRetry, ok := provReq.Spec.Parameters[NoRetryParameter]; ok && noRetry == "true" {
```

Comment:
Bordering on nit: it isn't great experience if invalid value for a known parameter is just quietly ignored.

I understand if you don't want to extend the scope of this CL, but I think for better UX in the future CA should inform the user that it's ignoring a supported parameter (at minimum by logging a warning that value is invalid), or better yet, fail the request with similar error.

Reply:
I agree, for this parameter I'll log if the value is invalid (the only valid value is `true` actually) however I believe CA should have more general mechanism for validating parameters. Hence I'll create an issue for that

Reply:
Thanks for pointing this out